Specs/DateTime/Dutch/DurationExtractor.json

```diff
@@ -688,10 +688,10 @@
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "voor de hele dag",
+        "Text": "de hele dag",
```

Comment:
All the changes in this PR don't make new cases pass? Only fix the timex in DateTimePeriod and change the "voor" behaviour?

Reply:
In the initial PR there were 29 remaining failing cases, this PR fixed those cases.

Reply:
I don't see them enabled in the specs in the PR. Only cases with "voor" and 4 other. Did I miss them?

Reply:
they seems to be there, e.g. lines 5733, 5859, 5911, 5937....

Reply:
Thanks. This is what I get for reviewing on a tablet. Missed an entire .json file...